dashboard/src/components/Skeleton/styles.css

```diff
@@ -0,0 +1,5 @@
+.remove-animate-pulse.animate-pulse {
+  animation: none !important;
```

Comment:
never use `!important` unless you have less than 30 minutes for the deadline of a task, or there is no other way around